foundry/run_node.rs

```diff
@@ -234,8 +234,13 @@ pub fn run_node(
 
     let time_gap_params = config.create_time_gaps();
 
-    let mut app_desc = AppDesc::from_str(&fs::read_to_string(&config.app_desc_path).unwrap()).unwrap();
-    app_desc.merge_params(&module_arguments).expect("error in merge params");
+    let mut app_desc = {
+        let app_desc_path = &config.app_desc_path;
+        let app_desc_string = fs::read_to_string(app_desc_path)
+            .map_err(|err| format!("Foundry failed to read app desc file from {}: {}", app_desc_path, err))?;
+        AppDesc::from_str(&app_desc_string).map_err(|err| format!("Foundry failed to parse app descriptor: {}", err))?
+    };
+    app_desc.merge_params(&module_arguments).map_err(|err| format!("Foundry failed to merge params {}", err))?;
```

Comment:
Maybe the message can be made more descriptive. `Foundry failed to merge params you supplied into the app descriptor.` should be better wording?

Reply:
I agree with you. Only saying "Merging params" does not give enough information.